README.md

```diff
@@ -124,10 +129,27 @@ marked as `unsupported`, which are ignored if found in the ZODB. If only a
 given object itself should be updated (properties, security settings etc.),
 `--no-recurse` can be used.
 
+### `zodbsync exec`
+
+This command requires the base directory to be a git repository and provides a
+wrapper for several git commands. It executes the given command, reads the
+objects changed between the old and new `HEAD` and plays them back. Any
+unstaged changes are stashed away and restored afterwards. The operation is
+aborted and rolled back if it results in a broken state (an interrupted
+`merge`, `rebase`, `cherry-pick` etc.) or if there is an overlap between the
+unstaged and the changed files.
+
+This allows commands like the following:
+
+    zodbsync exec "git cherry-pick COMMIT"
```

Comment:
I'm not sure whether this is possible, but it would be nice to be able to drop the quotes, and let "zodbsync exec" behave like other Unix exec wrappers, e.g. "timeout", "watch", and "flock". The examples would then read:

    zodbsync exec git cherry-pick COMMIT
    and so on.

Reply:
I do not think this is possible with `argparse`

Reply:
Alright, it was just a thought.